.travis.yml

```diff
@@ -9,9 +9,11 @@ go:
 before_install:
   - go get -u github.com/golang/lint/golint
   - if [[ $TRAVIS_GO_VERSION == 1.5* ]]; then go get -u github.com/kisielk/errcheck; fi
+  - go get -u golang.org/x/tools/cmd/vet
```

Comment:
Yes. The alternative is only to run `go vet` in 1.6, but minimizing dependence on `$TRAVIS_GO_VERSION` seems like a win.